content/docs/getting-started/quick-start.md

```diff
@@ -7,22 +7,33 @@ aliases:
 
 ## Installing
 
-Let's get started by installing Scratch Addons. Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+Let's get started by installing Scratch Addons.
+
+| Store | Install | Supported browsers |
+| - | - | - |
+| Chrome Web Store | [![Install for Chrome Web Store](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=install&color=4285F4)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco) | Google Chrome and other Chromium-based browser
+| Add-ons for Firefox | [![Install for Add-ons for Firefox](https://img.shields.io/amo/v/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=install&color=FF7139)](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/) | Mozilla Firefox
+| Microsoft Edge Addons | [![Install for Microsoft Edge Addons](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=microsoftedge&logoColor=white&label=install&color=0078D7&prefix=v&query=%24.version&url=https%3A%2F%2Fmicrosoftedge.microsoft.com%2Faddons%2Fgetproductdetailsbycrxid%2Filiepgjnemckemgnledoipfiilhajdjj)](https://microsoftedge.microsoft.com/addons/detail/iliepgjnemckemgnledoipfiilhajdjj) | Microsoft Edge
+
+More information can be read on [Installing](../installing).
 
-- [Chrome Web Store](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco) (for Google Chrome and other Chromium-based browsers)  
-- [Add-ons for Firefox](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/) (for Mozilla Firefox)  
-- [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/detail/iliepgjnemckemgnledoipfiilhajdjj) (for Microsoft Edge)  
 
 ## Getting started
 
 To get started, you need to access the settings page of the addon to see the functions of it.
 
-<!-- TODO: Add pictures -->
-
-1. Locate Scratch Addons on the toolbar and click it.  
+1. Locate Scratch Addons on the toolbar and click it.
+   
    Usually, it is on the top-right corner of your browser. For Google Chrome users, if you are not browsing Scratch, you need to click the puzzle icon to find the extension icon.
 
+   ![](/assets/img/getting-started/step-1.png)
+
 2. Press the gear icon to go to the Settings page.
 
-3. Feel free to look through the addons list and turn on/off whatever you want.  
-   You can also configure each of the addons on the bottom section of each addons.
\ No newline at end of file
+   ![](/assets/img/getting-started/step-2.png)
+
+3. Feel free to look through the addons list and turn on/off whatever you want.
+   
+   You can also configure each of the addons on the bottom section of each addons.

```

Comment:
> You can also configure each of the addons on the bottom section of each addons.

This sentence is taken from somewhere else I think, but it's phrased weirdly

Reply:
Actually I see it's been on this doc page forever so we could change it later too.

Reply:
Yeah, this is #171